Modules/mathmodule.c

```diff
@@ -1233,6 +1233,25 @@ FUNC2(remainder, m_remainder,
       "Return x - n*y where n*y is the closest integer multiple of y.\n"
       "In the case where x is exactly halfway between two multiples of\n"
       "y, the nearest even value of n is used. The result is always exact.")
+
+/*[clinic input]
+math.signbit
+
+    x: double
+    /
+
+Return True if 'x' is negative and False otherwise.
+
+This is useful to detect the sign bit of zeroes, infinities and NaNs.
```

Comment:
```suggestion
Return True if the sign of 'x' is negative and False otherwise.
```
I think we can be more verbose in sphinx docs.

Reply:
I actually used https://en.cppreference.com/w/c/numeric/math/signbit as a reference because I was too lazy to look at C11. Looking at it, I agree with this description.